src/main/java/mineverse/Aust1n46/chat/command/chat/Me.java

```diff
@@ -28,9 +28,9 @@ public boolean execute(CommandSender sender, String command, String[] args) {
 					msg = Format.FormatStringLegacyColor(msg);
 				}
 				if (sender.hasPermission("venturechat.color"))
-					msg = Format.FormatStringColor(msg);
+					msg = Format.FormatStringColor(msg, sender.hasPermission("venturechat.color.hex"));
 				if (sender.hasPermission("venturechat.format"))
-					msg = Format.FormatString(msg);
+					msg = Format.FormatString(msg, sender.hasPermission("venturechat.format.magic"));
```

Comment:
```suggestion
					msg = Format.FormatString(msg, sender.hasPermission("venturechat.format.obfuscated"));
```
Is the technical name for &k